crowdsourcing/servermgr/app.py

```diff
@@ -155,6 +168,16 @@ def status():
         return json.dumps({"progress": 90, "ip": ip})
 
     logging.info("status: success")
+    # register subdomain to proxy instance IP
+    if os.getenv("CLOUDFLARE_TOKEN"):
+        logging.info("registering subdomain on craftassist.io")
+        cloudflare_token = os.getenv("CLOUDFLARE_TOKEN")
+        cf = CloudFlare.CloudFlare(email='rebeccaqian@fb.com', token=cloudflare_token)
+        # TODO: zone ID is hard coded, should probably set in env var or get by zone name
+        zone_id = 'd2d53d14fffaecbfeb92e3e62f01607f'
```

Comment:
definitely remove this before committing this PR and instead power it through an env variable